src/main/kotlin/study/racinggame/race/reader/InputReader.kt

```diff
@@ -0,0 +1,5 @@
+package study.racinggame.race.reader
+
+fun interface InputReader {
+    fun raedLine(): String?
```

Comment:
```suggestion
    fun raedLine(): String
```

readln()을 사용하면 nullable 할 이유가 없겠죠~?